travelwise/travelwise/settings.py

```diff
@@ -134,10 +135,17 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.2/howto/static-files/
 
-STATIC_URL = 'static/'
+STATIC_URL = '/static/'
 STATICFILES_DIRS = [os.path.join(BASE_DIR, 'frontend/static')]
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
 
+#STORAGES = {
```

Comment:
Still need to figure out how to cache static files more efficiently. Will look back into this later

Reply:
You mean cache headers?